python/ray/data/_internal/execution/interfaces/execution_options.py

```diff
@@ -181,15 +181,28 @@ def min(self, other: "ExecutionResources") -> "ExecutionResources":
             ),
         )
 
-    def satisfies_limit(self, limit: "ExecutionResources") -> bool:
+    def satisfies_limit(
+        self,
+        limit: "ExecutionResources",
+        *,
+        exclude_object_store_memory=False,
```

Comment:
Why would we need this flag? 

To eval ability to launch the task we can ignore the object store, right?